packages/run-utils/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@microsoft/teamsfx-run-utils",
+  "version": "1.0.0",
+  "description": "Provides the utilities to run TeamsFx project locally.",
+  "main": "./lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "build": "rimraf lib && npx tsc -p ./",
```

Comment:
Consider later - use webpack to bundle all src and node_modules into one single file.

Reply:
Sure.